### DISCUSSION TITLE
Pake 打包跟網站內容有關嗎？有沒有可能透過 命令列參數提供網址？

### DISCUSSION THREAD

**Discussion by wade-fs** [tw93/Pake#304] [Q&A]
如題，
Pake 打包跟網站的內容有關嗎？
 如果無關的話，有沒有可能透過 命令列參數提供網址？

**Comment by tw93** [comment]
无关
有可能，你的意思是网址随便更改吗？
有使用pake-cli吗？这个是否满足需求
